modular_skyrat/modules/cortical_borer/code/cortical_borer_abilities.dm

```diff
@@ -78,8 +78,6 @@
 			cortical_owner.reagent_holder.reagents.add_reagent(reagent, cortical_owner.injection_rate_current, added_purity = 1)
 			cortical_owner.reagent_holder.reagents.trans_to(cortical_owner.human_host, cortical_owner.injection_rate_current, methods = INGEST)
 			var/obj/item/organ/internal/brain/victim_brain = cortical_owner.human_host.getorganslot(ORGAN_SLOT_BRAIN)
-			if(victim_brain)
-				cortical_owner.human_host.adjustOrganLoss(ORGAN_SLOT_BRAIN, 5)
```

Comment:
with this removal, you don't need the above variable.

Reply:
Fixed